### ISSUE TITLE
ss-broken class: broken links in content colours are not accessible

### ISSUE THREAD

**Issue by robbieaverill** [silverstripe/silverstripe-admin#850]
These are the styles we use at the moment for broken links in content fields:

```scss
// Editor variables
$state-danger-bg: #f2dede;
$state-danger-border: darken($state-danger-bg, 5%);
$text-color: #fff;

body.mceContentBody a.ss-broken { // sass-lint:disable-line class-name-format
  background-color: $state-danger-bg;
  border: 1px $state-danger-border solid;
  color: $text-color;
  padding: 1px;
  text-decoration: underline;
}
```

Firstly the TinyMCE class is now `mce-content-body`, but when you change the class this is what it looks like:

![image](https://user-images.githubusercontent.com/5170590/54796156-f7b1ea80-4cb3-11e9-8e9d-1e4945731c3c.png)

**Comment by robbieaverill** [comment]
cc @silverstripeux FYI I've just asked @sachajudd to sit with me while I change this to a `$brand-danger` background (#d40404) and white text. It passes WCAG AA. Please jump on the PR if you want it to be something else. This is essentially what was in CWP 1.x.

**Comment by robbieaverill** [comment]
PR at https://github.com/silverstripe/silverstripe-admin/pull/851

**Comment by ScopeyNZ** [comment]
Closed in #851
